docs/BackwardCompatibility.md

```diff
@@ -237,5 +242,6 @@ requirements<a name="1"></a>
 3. Kubernetes "[Serving multiple versions of a CRD](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#serving-multiple-versions-of-a-crd)"<a name="3"></a>
```

Comment:
The mentioned link presents another link: https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning/

Reply:
removed